content/applications/inventory_and_mrp/purchase/products/reordering.rst

```diff
@@ -1,96 +1,81 @@
-============================================
-Don’t run out of stock with reordering rules
-============================================
+==========================
+Configure reordering rules
+==========================
 
-To make sure you never run out of stock, you can define *Reordering Rules* on products. Thanks to
-them, Odoo can help you replenish your stock automatically when it reaches set quantities or
-whenever a sales order is created.
+For certain products, it is necessary to ensure that there is always a minimum amount available on
+hand at any given time. By adding a reordering rule to a product, it is possible to automate the
+reordering process so that a purchase order is automatically created whenever the amount on hand
+falls below a set threshold.
 
-.. important:: You need to install the *Inventory app* to use reordering rules.
+.. important::
+   The *Inventory* module must be installed to use reordering rules.
 
-Configure your storable product
-===============================
+Configure products for reordering
+=================================
 
-Open or create a product with its *Product Type* set to *Storable Product*.
+Products must be configured in a specific way before a reordering rule can be added to them. Create
+a new product by going to the *Inventory*, *Manufacturing*, *Purchase*, or *Sales* module and
+selecting :menuselection:`Products --> Products --> Create`. To allow for reordering, make sure
+that the :guilabel:`Can be Purchased` checkbox is activated and the :guilabel:`Product Type`
+dropdown is set to `Storable Product`.
```

Comment:
In this case I'd use `guilabel` tags here since we're implying to click/navigate on the UI, and `menuselection` doesn't quite work for a single item.

Small wording changes too to indicate where to `start` then then where to `navigate` after instead of `going to` and `selecting`.

```suggestion
a new product by starting in the :guilabel:`Inventory`, :guilabel:`Manufacturing`,
:guilabel:`Purchase`, or :guilabel:`Sales` module and then navigate to :menuselection:`Products -->
Products --> Create`. To allow for reordering, make sure that the :guilabel:`Can be Purchased`
checkbox is activated and the :guilabel:`Product Type` dropdown is set to `Storable Product`.
```